python_modules/airline-demo/airline_demo/types.py

```diff
@@ -20,10 +29,57 @@
 )
 
 
+class SparkDataFrameSerializationStrategy(SerializationStrategy):
```

Comment:
Why are we zipping the parquet file? The columns are already compressed (with snappy by default  think)